api/main.go

```
@@ -16,27 +19,53 @@ func main() {
+	testMode := os.Getenv("TEST_MODE") == "true"
+
+	var db *gorm.DB
+	var err error
+
+	if !testMode {
+		log.Println("Iniciando conexión a la base de datos...")
+		db, err = config.ConnectDB(config.NewDBConfig())
+		if err != nil {
+			log.Printf("Error al conectar con la base de datos: %v", err)
+			log.Println("Iniciando en modo de prueba sin base de datos...")
+			testMode = true
```

golangci-lint (1.64.8)

ineffectual assignment to testMode

(ineffassign)